aten/src/ATen/function_wrapper.py

```diff
@@ -1494,7 +1494,7 @@ def emit_body(env, option):
                                else ""
                 wrapped_tensor = CodeTemplate(ALLOC_WRAP[ret['type']]).substitute(
                     env, arguments=[call])
-                return_tensor = "return Tensor((${wrapped_tensor})${maybe_scalar},false);"
+                return_tensor = "return Tensor(c10::intrusive_ptr<TensorImpl, UndefinedTensor>::reclaim((${wrapped_tensor})${maybe_scalar}));"
```

Comment:
If you are feeling adventurous, I think the reclaim() here isn't strictly necessary either. The codegen changes might be involved though.